golos.publication/golos.publication.abi

```diff
@@ -231,7 +231,7 @@
       },
       {
         "name": "curators_prcnt",
-        "type": "uint16"
+        "type": "base_t"
```

Comment:
why `base_t`? there is no need to use it and `int64`, if we need such types, we can always convert to it inside contract